_posts/2024-02-07-black-history-spotlight-annie-easley.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: "US-RSE Black History Month Spotlight - Annie Easley"
+tags: [dei, black-history]
+author: Sophia Anyatonwu
+---
+
+US-RSE's [DEI working group (DEI-WG)](https://us-rse.org/wg/dei/)is proud to
+help US-RSE celebrate and participate in Black History Month. Each week during
+Black History Month, the US-RSE will spotlight Black/African Americans who have
+been involved in computing, science, engineering, and/or math, and have
+inspired our members through their accomplishments in their careers and their
+personal stories.
+
+## This week's Black History Month spotlight features Annie Easley
+
+{% include image.html
+url="https://image.cleveland.com/home/cleve-media/pgfull/img/plain-dealer/photo/2017/02/23/-8f07f7971aa2b185.JPG"
+description="Annie Easley works on a NASA computer. (Photo Courtesy of NASA)"
+style="float:right; padding:1em; width:350px" %}
+
+Some may know that the first computers in the US were women who were highly
+skilled in mathematics[^jpl]. One of these "human computers" was Annie Easley, a
+computer and rocket scientist who spent most of [her
+career](https://www.nasa.gov/general/annie-easley-computer-scientist/)
+contributing to and supporting key programs at the National Aeronautics and
+Space Administration (NASA)[^nasa].
+
+In 1955, Annie Easley began her career as a "human computer" at the National
+Advisory Committee for Aeronautics (NACA), the precursor to NASA, despite her
+initial plans to work in the healthcare field[^nasa][^encyclopedia].
+
+As human computers began to be replaced by machines, she developed the skills
+needed to become a programming expert and supported NASA programs in aerospace
+as well as green energy[^nasa][^encyclopedia]. Her contributions encompass
+working on alternative energy technologies, contributing to the development of
+the Centaur launch system, assessing how solar wind and energy could solve
+energy problems, and developing nuclear-powered rocket systems[^encyclopedia].
+
+Outside of being a computer and rocket scientist, Easley is known for her
+dedication to tutoring and encouraging students from underrepresented
+backgrounds to pursue STEM careers. She was also actively involved at NASA as
+an equal employment opportunity (EEO) counselor and advocate for improving
+employee morale in the workplace[^nasa][^encyclopedia].
+
+### References
```

Comment:
I don't think it's the result of anything you did explicitly, but the text for these anchors in the references is really small.  Do you know why that might be?

Reply:
There is an issue for this: https://github.com/USRSE/usrse.github.io/issues/1210